elasticsearch/src/test/java/akka/stream/alpakka/elasticsearch/ElasticsearchTest.java

```diff
@@ -39,8 +39,12 @@
 
   @BeforeClass
   public static void setup() throws IOException {
+    System.setProperty("es.set.netty.runtime.available.processors", "false");
```

Comment:
Could this obscure actual problems?

Reply:
Sorry, I'm still investigating and testing at my repository. Since this pull request is work in progress, I updated the subject.

For now, we (I and author of elasticsearch-cluster-runner library) think that actual problem is elasticsearch-cluster-runner can't run multiple cluster on same JVM at same time by internal structure of Elasticsearch. Maybe it can be resolved by merge Java test and Scala test.